src/libs/actions/Search.ts

```diff
@@ -42,7 +43,13 @@ Onyx.connect({
     waitForCollectionCallback: true,
 });
 
-function handleActionButtonPress(hash: number, item: TransactionListItemType | TransactionReportGroupListItemType, goToItem: () => void, isInMobileSelectionMode: boolean) {
+function handleActionButtonPress(
+    hash: number,
+    item: TransactionListItemType | TransactionReportGroupListItemType,
+    goToItem: () => void,
+    isInMobileSelectionMode: boolean,
+    currentSearchKey?: SuggestedSearchKey,
```

Comment:
```suggestion
    currentSearchKey: SuggestedSearchKey | undefined,
```

This seems to be a required parameter, let's make sure it's passed

(do same with other functions that introduced this param)